staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -44,7 +45,7 @@ var (
 			Help:           "Number of LIST requests served from watch cache",
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
-		[]string{"resource_prefix", "index"},
+		[]string{"group", "resource", "index"},
```

Comment:
Same question about changing labels here.
@dgrisonnet

Reply:
I am fine with it as long as it is mentioned in the changelog